scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1251,6 +1237,27 @@ class KubernetesMonitor( ScalyrMonitor ):
 
     """
 
+    # A map of param to
+    ENVIRONMENT_AWARE_CONFIG_PARAMS = {
+        'container_check_interval': None,
+        'docker_max_parallel_stats': None,
+        'container_globs': None,
+        'report_container_metrics': None,
+        'report_k8s_metrics': None,
+        'k8s_ignore_pod_sandboxes': None,
+        'k8s_include_all_containers': None,
+        'k8s_use_v2_attributes': None,
+        'k8s_use_v1_and_v2_attributes': None,   # echee TODO (double check w steven)
```

Comment:
Need input from @czerwingithub whether this param needs to be environment-aware

Background notes from Imron: 
v1 did not group by deployments. In order to prevent breakages, this flag was added so that customers who had dashboards that relied on the old attribute names would still work.

Reply:
The conclusion during today's standup:

k8s_use_v2_attributes & k8s_use_v1_and_v2_attributes don't need to be environment-aware.  Reasoning: users of k8s config maps are likely on Starbuck/v2 anyways.